schemas/com.mbta.ctd.glides.operator_signed_in.v1.json

```diff
@@ -28,6 +28,10 @@
         "type": {
           "type": "string",
           "examples": ["tap", "type"]
+        },
```

Comment:
question: I missed this before, so this might be more of a question for @skyqrose, but this schema doesn't enforce that `type` is required, which doesn't appear to match the textual description above.

Reply:
Oh, I think that's fine to be optional, it's still meaningful to say an operator signed in even if you don't know how they signed. But it does seem like an oversight that `event.data.signature` and `event.data.signedInAt` are required.
Preston, can you fix the schema to make `signature` and `signedInAt` required, and update the docs to make sure that things all the signature data is correctly labeled as required or optional?

Reply:
Yep!